src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityTests.java

```diff
@@ -139,54 +233,42 @@ void assertBasicSearchWorks() {
         ElasticsearchAssertions.assertNoFailures(searchReq.get());
     }
 
-    void assertRealtimeGetWorks() {
-        assertAcked(client().admin().indices().prepareUpdateSettings("test").setSettings(ImmutableSettings.builder()
+    void assertRealtimeGetWorks(String indexName) {
+        assertAcked(client().admin().indices().prepareUpdateSettings(indexName).setSettings(ImmutableSettings.builder()
             .put("refresh_interval", -1)
             .build()));
-        SearchRequestBuilder searchReq = client().prepareSearch("test").setQuery(QueryBuilders.matchAllQuery());
+        SearchRequestBuilder searchReq = client().prepareSearch(indexName).setQuery(QueryBuilders.matchAllQuery());
         SearchHit hit = searchReq.get().getHits().getAt(0);
         String docId = hit.getId();
         // foo is new, it is not a field in the generated index
-        client().prepareUpdate("test", "doc", docId).setDoc("foo", "bar").get();
-        GetResponse getRsp = client().prepareGet("test", "doc", docId).get();
+        client().prepareUpdate(indexName, "doc", docId).setDoc("foo", "bar").get();
+        GetResponse getRsp = client().prepareGet(indexName, "doc", docId).get();
         Map<String, Object> source = getRsp.getSourceAsMap();
         assertThat(source, Matchers.hasKey("foo"));
 
-        assertAcked(client().admin().indices().prepareUpdateSettings("test").setSettings(ImmutableSettings.builder()
+        assertAcked(client().admin().indices().prepareUpdateSettings(indexName).setSettings(ImmutableSettings.builder()
             .put("refresh_interval", "1s")
```

Comment:
I changed the value to use the constant, since that could change, but I think using the actual parameter inline makes it much easier to read through test code than using constants everywhere.